marshmallow/utils.py

```diff
@@ -317,11 +317,38 @@ def to_iso_date(date, *args, **kwargs):
     return datetime.date.isoformat(date)
 
 
+def from_timestamp(timestamp, tzinfo=UTC):
+    return datetime.fromutctimestamp(float(timestamp)).replace(tzinfo=tzinfo)
```

Comment:
Per the [documentation](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromtimestamp), `datetime.fromtimestamp(value, tz=datetime.timezone.utc)` is preferred over `datetime.fromutctimestamp(value)`.